### PR TITLE
Added Dining Header and Campus Express Login

### PR DESCRIPTION
**Pull request by aantenberg**

### CODE REVIEW

**Review by 01jongmin — CHANGES_REQUESTED** [review]
Just a couple of changes!

File: PennMobile/Dining/Model/DiningToken.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+//
+
+import Foundation
+struct DiningToken: Codable {
+    let value: String
+    var expiration: Int {
+        didSet {
+            expirationDate = Date().advanced(by: .init(expiration))
+        }
+    }
```
[the remark this answers is not in the document]
Comment (01jongmin) [reply PennMobile/Dining/Model/DiningToken.swift:16]: Got it. Can we then set expiration to be private as we won't be using it

[the remark this answers is not in the document]
Comment (aantenberg) [reply PennMobile/Dining/Model/DiningToken.swift:16]: Yup sounds good!

**Review by 01jongmin — APPROVED** [review]
LGTM!

**Pull request merged**